addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -92,14 +92,14 @@ def action_quit(self):
         self.ensure_one()
         return self.write({'user_ids': [(3, self._uid)]})
 
-    def action_view_rating(self):
+    def action_view_channel_ratings(self):
         """ Action to display the rating relative to the channel, so all rating of the
             sessions of the current channel
-            :returns : the ir.action 'action_view_rating' with the correct domain
+            :returns : the ir.action 'action_view_channel_ratings' with the correct context
         """
         self.ensure_one()
         action = self.env['ir.actions.act_window']._for_xml_id('im_livechat.rating_rating_action_view_livechat_rating')
-        action['domain'] = [('parent_res_id', '=', self.id), ('parent_res_model', '=', 'im_livechat.channel')]
+        action['context'] = {'search_default_parent_res_name': self.name}
```

Comment:
Isn't it possible to use id instead of name here?

Reply:
No filteration is not working with `id`. I tried.

Reply:
Hello,

You need to add the field in the search view of `rating.rating` to be able to use it with a `search_default_`.

`<field name="parent_res_id" filter_domain="[('parent_res_id','ilike',self)]"/>`

Then it's possible to use id here, which is more robust than name.

Reply:
Hello,
As per your this suggestion, I tried and yes it does filteration with ID.

But we need to do same as project rating do as per @LNA's suggestion. In project, default search is done with 'parent document name(name)'(https://drive.google.com/file/d/1GoP_Cb-7xSE44ZyJdkEGB0EiSI4JY6ZS/view?usp=sharing) and as we did filteration with id here so it shows 'parent document(id)'(https://drive.google.com/file/d/1nLW6GhJt9clorlIEoE5_QfjfxPP0YE7G/view?usp=sharing) in header thus its not the same as project rating. May be we can change the string(parent document) but can't change the value(Id).

Reply:
I will ask him on the task, thanks for the feedback.

Reply:
Hello @seb-odoo 

As per CPE's opinion on the task (`parent document name(channel name) is more understandable when filtering the ratings`
Updated commit accordingly. Thanks.

Reply:
Ok, thank you for considering it and for the update.